components/database/couchdb-31/couchdb-common.p5m

```diff
@@ -0,0 +1,26 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
+#
+
+set name=pkg.fmri value=pkg:/database/$(COMPONENT_NAME)-common-3@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```

Comment:
Are there any reasons to avoid calling package just couchdb-common? I mean, if/when couchdb 4 is released, should it have separate user?

Reply:
couchdb-common is obsoloted.
It should be resurrected but I have no idea how to manage that.

Reply:
Yes, remove from history and ensure that new version > old one